internal/auth.go

```diff
@@ -16,41 +16,54 @@ import (
 	"github.com/thomseddon/traefik-forward-auth/internal/provider"
 )
 
+const ERRORS_COOKIE_HAS_EXPIRED = "cookie has expired"
+const ERRORS_UNABLE_TO_PARSE_COOKIE_EXPIRY = "unable to parse cookie expiry"
+const ERRORS_INVALID_COOKIE_MAC = "invalid cookie mac"
+const ERRORS_UNABLE_TO_GENERATE_MAC = "unable to generate mac"
+const ERRORS_UNABLE_TO_DECODE_MAC = "unable to decode cookie mac"
+const ERRORS_INVALID_COOKIE_FORMAT = "invalid cookie format"
+
+const ERRORS_INVALID_CSRF_COOKIE_VALUE = "invalid CSRF cookie value"
+const ERRORS_INVALID_CSRF_STATE_VALUE = "invalid CSRF state value"
+const ERRORS_CSRF_COOKIES_DOESNT_MATCH = "csrf cookie does not match state"
+const ERRORS_INVALID_CSRF_FORMAT = "invalid CSRF state format"
```

Comment:
They should probably start with a capital letter

Reply:
youre right

Reply:
Thank you anyways for the comment 🌟